desktop/src/app/components/navbar/warnings/modals/delete-resource-modal.component.ts

```diff
@@ -40,6 +43,20 @@ export class DeleteResourceModalComponent {
         if (event.key === 'Escape') this.activeModal.dismiss('cancel');
     }
 
+    public async initialize() {
+
+        const foundDocuments: Array<Document> = await this.datastore.find(
+            { categories: ['UNCONFIGURED'] },
+            { includeResourcesWithoutValidParent: true }
+        ).then(res => res.documents);
```

Comment:
Code style: "then" should only be used when absolutely necessary, so this should be replaced with something like "(await this.datastore.find(...)).documents"